api/src/main/java/com/redhat/ipaas/api/v1/rest/util/PaginationOptions.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.ipaas.api.v1.rest.util;
+
+public interface PaginationOptions {
+
+    int getPage();
```

Comment:
Would prefer something that can be part of maven build for now for easy use during development. In my experience, sonarqube doesn't add much & it takes time to tune metrics to actually be useful rather than overly noisy. Happy for someone (you?) to look at it though & see what it looks like for us?